packages/docusaurus/lib/load/plugins.js

```diff
@@ -20,8 +20,8 @@ module.exports = async function loadPlugins({pluginConfigs = [], context}) {
       try {
         // eslint-disable-next-line
         Plugin = require(name);
-      } catch (e) {
-        throw new Error(`'${name}' plugin cannot be found.`);
+      } catch (ex) {
+        throw new Error(`Error loading '${name}' plugin.`);
```

Comment:
We shouldn't be swallowing the exception and just giving our own ambiguous error message here. Rethrowing the thrown error message will actually be more useful.